interface-routes/elevate-mentoring.json

```diff
@@ -4507,6 +4506,19 @@
                     "packageName": "elevate-mentoring"
                 }
             ]
+        },
+		{
+            "sourceRoute": "/mentoring/health",
```

Comment:
@MallanagoudaB  add routes changes to this develop branch 
https://github.com/ELEVATE-Project/utils/blob/master/interface-routes/elevate-qa-routes.json